app.js

```diff
@@ -4,12 +4,22 @@ var express = require('express');
 var app = express();
 var debug = require('debug')('*');
 
+var emailApi = require('./controller/email');
+var templateApi = require('./controller/template');
+
 app.use(require('body-parser').urlencoded({extended: true}));
 app.use(require('body-parser').json());
 
-app.use(function errorHandler(err, req, res) {
-  debug('err %o, req %o, res %o', err, req, res);
-});
+// Email api endpoints
+app.post('/email/send', emailApi.sendEmail);
```

Comment:
this'll work fine but its fairly standard to keep routes separate i.e., `app.use(require('./routes'))`